workflows/wdl/starfish.wdl

```diff
@@ -0,0 +1,100 @@
+
+task process_field_of_view {
+
+    String experiment
+    Int field_of_view
+    String recipe_file
+
+    command <<<
+        wget -O recipe.py ${recipe_file}
+
+        python3 <<CODE
+
+        import starfish
+        recipe = __import__('recipe')
+
+        decoded_spots = recipe.process_fov(${field_of_view}, "${experiment}")
+        decoded_spots.save_csv("decoded.csv")
+
+        CODE
+    >>>
+
+    runtime {
+        docker: "spacetx/starfish:0.1.0-simple"
+        memory: "16 GB"
+        cpu: "4"
+        disk: "local-disk 100 SDD"
+    }
+
+    output {
+        File decoded_csv = "decoded.csv"
+    }
+}
+
+
+task concatenate_fovs {
+    Array[File] decoded_csvs
+
+    command <<<
+        python <<CODE
+        files = "${sep=' ' decoded_csvs}".strip().split()
```

Comment:
it's kind of concerning that this is a bunch of python code floating in the ether.

Reply:
Yea I thought about moving this in starfish and making a wdl_utils file? But I thought that might be kind of random if it was the only wdl related thing...down to do that though if you think it makes more sense